bin/run-tests

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Running tests in using test config 'docker-compose.test.yml'"
+
+docker-compose \
+    -f 'docker-compose.test.yml' \
+    run test-backend bash -c "yarn db:migrate; yarn db:seed; yarn test:ci"
```

Comment:
Should we use `&&` instead of just `;`?

Reply:
The difference between `;` and `&&` is that with `&&` subsequent commands will not run if they return a non-zero exit code, which is usually what we want. However, if someone were to manually create/migrate/seed the db (if they had reason to) then this script would still get to the test running stage. Maybe since this script is for convenience, that's a use case we don't want to cover?